ansible/roles-infra/infra-ec2-template-generate/templates/cloud_template.j2

```diff
@@ -250,6 +250,9 @@ Resources:
             VolumeType: "{{ vol[cloud_provider].type }}"
           {% else %}
             VolumeType: "{{ aws_default_volume_type }}"
+          {% endif %}
+          {% if vol.snapshop is defined %}
```

Comment:
typo here. snapshot, not snapshop

Let's use `snapshot_id` to be explicit 

`instances[].volumes[].snapshot_id`